### ISSUE TITLE
Handling message Open to get notification contents

### ISSUE THREAD

**Issue by cdmunoz** [la-haus/iterable-flutter#2]
Currently we haven't found a way to handle when a push notification is clicked/tapped in order to extract custom data from it.

## One Signal Example
With our current push notifications provider, they offer in the SDK a way to handle when a push notification is Opened:

```dart
OneSignal.shared.setNotificationOpenedHandler((OSNotificationOpenedResult result) {
          // Use payload to open different views
    });
```

## Firebase Example
Even, Firebase Messaging has a similar way to do it:

```dart
FirebaseMessaging.onMessageOpenedApp.listen((RemoteMessage message) {
          // Use payload to open different views
    });
```

**Comment by danielgomezrico** [comment]
@cdmunoz did we already fixed this on https://github.com/la-haus/iterable-flutter/releases/tag/v0.1?

**Comment by cdmunoz** [comment]
Yes! it's already fixed indeed with [v0.1](https://github.com/la-haus/iterable-flutter/releases/tag/v0.1)
